core/src/main/java/org/nervos/ckb/service/NervosApi.java

```diff
@@ -0,0 +1,28 @@
+package org.nervos.ckb.service;
+
+import org.nervos.ckb.request.Request;
+import org.nervos.ckb.response.*;
+import org.nervos.ckb.response.item.Cell;
+
+/**
+ * Created by duanyytop on 2018-12-20.
+ * <p>
+ * Copyright © 2018 Nervos Foundation. All rights reserved.
+ */
+public interface NervosApi {
```

Comment:
I feel we should use `CKB` prefix instead of `Nervos`.

Reply:
OK